Report/Creatable/index.ts

```diff
@@ -1,9 +1,10 @@
+import { Project as CreatableProject } from "./Project"
 import { Salary as CreatableSalary } from "./Salary"
 
-export type Creatable = Creatable.Salary
+export type Creatable = Creatable.Project | Creatable.Salary
```

Comment:
with this change the cost type is only pointing on the creatable salary type. we need to update the const to point to a union of both salary and project type